docs/content/install/install-on-google-cloud-run.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Install on Google Cloud Run"
+date: 2019-10-10T18:16:43+11:00
+draft: false
+weight: 4
+---
+
+[Google Cloud Run](https://cloud.google.com/run/) is a service that aims to bridge the gap between the maintainance benefits of serverless architecture and the flexibility of Kubernetes. It is built on top of the opensource [Knative](https://knative.dev/) project. Deploying using Cloud Run is similar to deploying using [Google App Engine](/install/install-on-gae) with the benefits of a free tier and a simpler build process. 
+
+## Selecting a Storage Provider
+
+There is documentaion about how to use environment variables to configure a large number of storage providers; however, for this prarticular example we will use [Google Cloud Storage](https://cloud.google.com/storage/)(GCS) because it fits nicely with Cloud Run.
+
+## Before You Begin
+
+This guide assumes you have completed the following tasks:
+
+- Signed up for Google Cloud
+- Installed the [gcloud](https://cloud.google.com/sdk/install) command line tool 
+- Installed the beta plugin for ghe gcloud command line tool ([this is how to set it up](https://cloud.google.com/run/docs/setup))
+- Created a (GCS) bucket for your go modules
+
+## Setup
+
+First set up configuration environment variables
+
+```bash
+export GOOGLE_CLOUD_PROJECT=your-project
+export ATHENS_REGION=asia-northeast1
+export GCS_BUCKET=your-bucket-name
+```
+
+You will then need to push a copy of the Athens docker image to your google cloud container registry.
+
+```bash
+docker pull gomods/athens:v0.6.0 
+
+docker tag gomods/athens:v0.6.0 gcr.io/$GOOGLE_CLOUD_PROJECT/athens:v0.6.0
+
+gcloud builds submit --tag gcr.io/$GOOGLE_CLOUD_PROJECT/athens:v0.6.0
+```
```

Comment:
Can you put a `?` character before each command? We do that pattern elsewhere in the documentation to indicate that these are commands that the user should run.

Reply:
@arschles I am a little confused about this. Do you mean the line should be? 

```console
? gcloud builds submit --tag gcr.io/$GOOGLE_CLOUD_PROJECT/athens:v0.6.0
```